src/components/functional/FloatingArrow.tsx

```diff
@@ -0,0 +1,60 @@
+import { FunctionalComponent, h } from "@stencil/core";
+import { JSXAttributes } from "@stencil/core/internal";
+import { FloatingLayout } from "../../utils/floating-ui";
+
+interface FloatingArrowProps extends JSXAttributes {
+  floatingLayout: FloatingLayout;
+}
+
+const CSS = {
+  arrow: "calcite-floating-ui-arrow",
+  arrowStroke: "calcite-floating-ui-arrow__stroke"
+};
+
+const DEFAULTS = {
+  width: 12,
+  height: 6,
+  strokeWidth: 1
+};
+
+export const FloatingArrow: FunctionalComponent<FloatingArrowProps> = ({
```

Comment:
Can you add some doc on the usage of this component? This should reference `floating-ui.ts` and viceversa.